.github/workflows/conformance-externalworkloads.yaml

```diff
@@ -1,484 +0,0 @@
-name: Conformance External Workloads (ci-external-workloads)
-
-# Any change in triggers needs to be reflected in the concurrency group.
-on:
-  workflow_dispatch:
-    inputs:
-      PR-number:
-        description: "Pull request number."
-        required: true
-      context-ref:
-        description: "Context in which the workflow runs. If PR is from a fork, will be the PR target branch (general case). If PR is NOT from a fork, will be the PR branch itself (this allows committers to test changes to workflows directly from PRs)."
-        required: true
-      SHA:
-        description: "SHA under test (head of the PR branch)."
-        required: true
-      extra-args:
-        description: "[JSON object] Arbitrary arguments passed from the trigger comment via regex capture group. Parse with 'fromJson(inputs.extra-args).argName' in workflow."
-        required: false
-        default: '{}'
-  push:
-    branches:
-      - 'renovate/main-**'
-  # Run every 8 hours
-  schedule:
-    - cron:  '0 4/8 * * *'
-
-# By specifying the access of one of the scopes, all of those that are not
-# specified are set to 'none'.
-permissions:
-  # To read actions state with catchpoint/workflow-telemetry-action
-  actions: read
-  # To be able to access the repository with actions/checkout
-  contents: read
-  # To allow retrieving information from the PR API
-  pull-requests: read
-  # To be able to set commit status
-  statuses: write
-  # To be able to request the JWT from GitHub's OIDC provider
-  id-token: write
-
-concurrency:
-  # Structure:
-  # - Workflow name
-  # - Event type
-  # - A unique identifier depending on event type:
-  #   - schedule: SHA
-  #   - workflow_dispatch: PR number
-  #
-  # This structure ensures a unique concurrency group name is generated for each
-  # type of testing, such that re-runs will cancel the previous run.
-  group: |
-    ${{ github.workflow }}
-    ${{ github.event_name }}
-    ${{
-      (github.event_name == 'push' && github.sha) ||
-      (github.event_name == 'schedule' && github.sha) ||
-      (github.event_name == 'workflow_dispatch' && github.event.inputs.PR-number)
-    }}
-  cancel-in-progress: true
-
-env:
-  clusterName: ${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}-${{ github.run_attempt }}
-  vmName: ${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}-${{ github.run_attempt }}
-  vmStartupScript: .github/gcp-vm-startup.sh
```

Comment:
While at it maybe also remove `.github/gcp-vm-startup.sh` which only seems to be used by this workflow.

Reply:
Thanks!